app/controllers/proposals_controller.rb

```diff
@@ -99,7 +99,7 @@ def parse_edit_field
   def proposal_params
     params.require(:proposal).permit(:title, {tags: []}, :session_format_id, :track_id, :abstract, :details, :pitch, custom_fields: @event.custom_fields,
                                      comments_attributes: [:body, :proposal_id, :user_id],
-                                     speakers_attributes: [:bio, :user_id, :id])
+                                     speakers_attributes: [:bio, :event_id, :user_id, :id])
```

Comment:
Let's remove the event_id from speaker_attributes on params and explicitly set it in create/update to whatever @event is.  It's possible that someone could form hack and submit a different event_id.   Since we've already got @event and use that for proposal create/update, let's enforce on the speaker update as well.